src/.vuepress/components/JavaDeprecatedBanner.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="alert alert-warning deprecatedBanner">
+    <p>
+      This SDK is deprecated, we advise you to use from now on the Kuzzle
+      SDK-JVM.<br />
```

Comment:
```suggestion
      This SDK is deprecated. We recommend to use the Kuzzle SDK-JVM.<br />
```
Link to the jvm sdk missing also?